remoteappmanager/application.py

```diff
@@ -23,7 +23,7 @@ class Application(web.Application, LoggingMixin):
 
     user = Instance(User, allow_none=True)
 
-    db = Instance(orm.Database, allow_none=True)
+    db = Instance(ABCAccounting, allow_none=True)
```

Comment:
Ok, at this point, I think that the `db` name is deceiving. Also, what happens when we decide to handle more tasks than just accounting in the database?